observation_portal/observations/viewsets.py

```diff
@@ -156,12 +168,30 @@ def create(self, request, *args, **kwargs):
             cache.set(cache_key + f"_{site}", timezone.now(), None)
             return Response({'num_created': len(observations), 'errors': errors}, status=201)
 
+    def get_request_serializer(self, *args, **kwargs):
```

Comment:
For viewsets with multiple actions/serializers I've adopted this pattern of defining a dict keyed by action and returning the proper serializer based on the action. It works well and is pretty clean, though I admit it kind of violates DRY, as we're almost always doing a `serializers.get()` with a sensible default - but since these are override method I didn't want to assume too much about what the implementation will look like for every view just to keep things flexible. I can probably break this stuff out into a mixin if we deem that necessary, though.